client/src/Components/shared/AppBarCollapse/AppBarCollapse.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import ButtonAppBarCollapse from "./ButtonAppBarCollapse";
+import MenuList from "./MenuList";
+import CollapsedMenuList from "./CollapsedMenuList";
+
+const useStyles = makeStyles((theme) => ({
+  root: {},
+  buttonBar: {
+    [theme.breakpoints.down("xs")]: {
+      display: "none",
+    },
+    display: "flex",
+    alignItems: "center",
+  },
+}));
+
+const AppBarCollapse = (props) => {
```

Comment:
```suggestion
const AppBarCollapse = ({isAuthenticated}) => {
```

Reply:
if you're able to use this syntax, it's a great way to keep track of the object `props` and see what is actually being utilized